.circleci/config.yml

```diff
@@ -1,123 +1,29 @@
-    default: false
-  dev-orb-version:
-    type: string
-    default: "dev:beta"
-
-
-# --------------
-# Pipeline tasks
-# --------------
-
-jobs:
-
-  integration-test_python-lint-package:
-    executor: noos-ci/default
-    steps:
-      - checkout
-      - run:
-          name: Copy fixtures files over
-          command: cp -R ./fixtures/python/* .
-      - noos-ci/python-lint-package:
-          pkg-manager: pipenv
-          install-args: "--dev"
-          configure-git: false
-          src-directory: "./src"
-
-  integration-test_python-test-package:
-    executor: noos-ci/default
-    steps:
-      - checkout
-      - run:
-          name: Copy fixtures files over
-          command: cp -R ./fixtures/python/* .
-      - noos-ci/python-test-package:
-          pkg-manager: pipenv
-          install-args: "--dev"
-          configure-git: false
-          tests-directory: "./src/tests"
-
-  integration-test_python-build-wheel:
-    executor: noos-ci/default
-    steps:
-      - checkout
-      - run:
-          name: Copy fixtures files over
-          command: cp -R ./fixtures/python/* .
-      - noos-ci/python-build-wheel:
-          pkg-manager: pipenv
-          install-args: "--dev"
-          configure-git: false
-
-  integration-test_helm-lint-chart:
-    executor: noos-ci/default
-    steps:
-      - checkout
-      - noos-ci/helm-lint-chart:
-          chart-directory: ./fixtures/helm
+  orb-tools: circleci/orb-tools@12.0.4
 
-  integration-test_helm-build-chart:
-    executor: noos-ci/default
-    steps:
-      - checkout
-      - noos-ci/helm-build-chart:
-          registry-provider: aws
-          chart-directory: ./fixtures/helm
-          chart-name: noos-prod/noos-test
-          chart-tag: 0.0.1
-
-  integration-test_docker-build-image:
-    executor: noos-ci/default
-    steps:
-      - checkout
-      - noos-ci/docker-build-image:
-          registry-provider: docker
-          configure-git: false
-          image-context: ./fixtures/docker
-          image-name: busybox
-          image-tag: ${CIRCLE_SHA1}
-
-  integration-test_terraform-run-plan:
-    executor: noos-ci/default
-    steps:
-      - checkout
-      - noos-ci/terraform-update-variable:
-          workspace: noos-prod
-          variable-name: noos_ci_integration_test
-          variable-value: ${CIRCLE_SHA1}
-      - noos-ci/terraform-run-plan:
-          workspace: noos-prod
-          plan-message: ${CIRCLE_BUILD_URL}
 
 
 # -----------------
 # Pipeline workflow
 # -----------------
 
-# https://circleci.com/docs/2.0/creating-orbs/#integration-test_deploy
-# https://circleci.com/developer/orbs/orb/circleci/orb-tools
+# https://github.com/CircleCI-Public/Orb-Template/blob/main/.circleci/config.yml
+# https://circleci.com/docs/testing-orbs/
 
 workflows:
 
   # This workflow will run on every commit
   lint-build:
-    unless: << pipeline.parameters.run-integration-tests >>
     jobs:
       - orb-tools/lint
       - orb-tools/pack
```

Comment:
:question_mark:

Is there a reason not adding extra sanity checks, such as the `orb-tools/review` job?

https://circleci.com/developer/orbs/orb/circleci/orb-tools#jobs-review